node-wasm/js/README.md

```diff
@@ -0,0 +1,19 @@
+# Lumina node wasm
+
+A compatibility layer for the [`Lumina`](https://github.com/eigerco/lumina) node to
+work within a browser environment and be operable with javascript.
+
+# Example
+Starting lumina inside a dedicated worker
+
+```javascript
+import { spawnNode, Node, NodeConfig, Network } from "lumina-node";
+
+const node = await spawnNode();
+const mainnetConfig = NodeConfig.default(Network.Mainnet);
+
+await node.start(mainnetConfig);
+
+await node.wait_connected();
+await node.request_head_header();
+```
```

Comment:
I think we should mention there that if someone doesn't use the `spawnNode` then he must call init to initialize wasm module, which is a default export.

Reply:
Added note and example below 👍